src/domain/space/space.defaults/definitions/subspace.community.application.form.ts

```diff
@@ -1,33 +1,31 @@
 import { CreateFormInput } from '@domain/common/form/dto/form.dto.create';
 
 export const subspceCommunityApplicationForm: CreateFormInput = {
-  description: '',
+  description: '<strong>Welcome!</strong>\ Thank you for expressing interest in joining our community. We would like to learn more about you and how you can contribute. Please take a moment to complete the following application form.',
```

Comment:
I think the slash after the strong closing tag should be removed:
`<strong>Welcome!</strong>\`  -> `<strong>Welcome!</strong>`

Reply:
Should I use a <br> instead? I took this from the response in the inspector panel when I created a post with the desired text 🙃

Reply:
Just remove the backslash otherwise you'll get:
"**Welcome!**\ Thank you for expressing interest in joining our community. We would like to learn more about you and how you can contribute. Please take a moment to complete the following application form."